.gitignore

```diff
@@ -18,3 +18,7 @@ dist/
 .eggs/
 .tox/
 .venv/
+
+# local files
```

Comment:
Do not ignore thoses.

Reply:
Fixed it, not ignoring the failure.xml file anymore. The other one isn't needed, I was using it to keep the original unchanged so I didn't have to recreate it.